java/pom.xml

```diff
@@ -1099,13 +1101,14 @@
       <properties>
         <arrow.dataset.jni.dist.dir>java-dist</arrow.dataset.jni.dist.dir>
         <cpp.dependencies.builded>false</cpp.dependencies.builded>
-        <ARROW_CSV>ON</ARROW_CSV>
-        <ARROW_ORC>OFF</ARROW_ORC>
+        <ARROW_DATASET>ON</ARROW_DATASET>
+        <ARROW_GANDIVA>OFF</ARROW_GANDIVA>
```

Comment:
I think ARROW_GANDIVA is accidentally set to OFF here

Reply:
Or is it not supported on Windows?

Reply:
Doesn't have JNI yet because we need to install LLVM to build Gandiva ([Could NOT find LLVM (missing: LLVM_DIR)](https://github.com/apache/arrow/pull/14203#issuecomment-1257161265)).